source/agora/node/Validator.d

```diff
@@ -383,6 +383,8 @@ public class Validator : FullNode, API
 
     protected override void acceptHeader (const(BlockHeader) header) @safe
     {
+        // First we must validate the header
+        this.ledger.validateBlockSignature(header);
```

Comment:
We don't do anything with the return value though ?

Reply:
Updated